diesel/src/query_dsl/mod.rs

```diff
@@ -1,3 +1,12 @@
+//! Traits with methods to construct select statements
```

Comment:
Hm. Can we find something other than "Traits with methods to" to say this is a thing that contains things to do select things? Maybe

> Module with traits to construct SELECT statements